slack_sdk/socket_mode/logger/messages.py

```diff
@@ -0,0 +1,6 @@
+import re
+
+
+def debug_message_redact(message: str) -> str:
```

Comment:
It seems that you followed this file and method naming: https://github.com/slackapi/bolt-python/blob/main/slack_bolt/logger/messages.py The format is more of "{log_level}_{log meaning}", thus "debug_redacted_message_string" or something like that would be even better. What do you think?
```suggestion
def debug_redacted_message_string(message: str) -> str:
```

Reply:
Agreed 👍 I like `debug_redacted_message_string`